recipes/tributary/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "tributary" %}
+{% set version = "0.1.0rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 11d15f1eaeeb6c27247dc7706175334aaee06cc9b023ea4a8cc107e220a4b877
+
+build:
+  number: 0
+  skip: True  # [py<37 or (win)]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - aiohttp >=3.5.4
+    - aiofiles >=0.4.0
+    - future >=0.17.1
+    - gevent >=1.3.7
+    - graphviz >=0.10.1
+    - ipython >=7.0.1
+    - ipywidgets >=7.5.1
+    - networkx >=2.2
+    - numpy >=1.15.3
+    - pandas >=0.19.0
+    - python-confluent-kafka >=0.11.6
```

Comment:
The wheel on pypi is a bit more strict:
```
Requires-Dist: confluent-kafka (==0.11.6)
```
Just noting it down here, in case that's of concern.

Reply:
good spot:
https://github.com/timkpaine/tributary/commit/c9e41b1c1dbcfe53a8294b04a5dfb4e99ed78c2b